.github/workflows/ci_cron_weekly.yml

```diff
@@ -0,0 +1,118 @@
+name: Weekly cron
+
+# This is weekly cron on astropy dev but not here because
+# release branch is low traffic. But we keep the file as close
+# to dev version as possible for ease of backport.
+on:
+  push:
+    branches:
+      - v4.2.x
+  pull_request:
+    branches:
+      - v4.2.x
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    if: github.repository == 'astropy/astropy'
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          # DISABLED FOR RELEASE BRANCH (is a duplicate when not cron)
+          # We check numpy-dev also in a job that only runs from cron, so that
+          # we can spot issues sooner. We do not use remote data here, since
+          # that gives too many false positives due to URL timeouts. We also
+          # install all dependencies via pip here so we pick up the latest
+          # releases.
+          #- name: Python 3.7 with dev version of key dependencies
+          #  os: ubuntu-latest
+          #  python: 3.7
+          #  toxenv: py37-test-devdeps
+
+          - name: Documentation link check
+            os: ubuntu-latest
+            python: 3.7
+            toxenv: linkcheck
```

Comment:
Maybe we don't need linkcheck for release branches either?

Reply:
I think we can remove it.

Reply:
It is commented out.